### ISSUE TITLE
Extends RPC methods

### ISSUE THREAD

**Issue by 0xslipk** [ConsenSys-archive/ganache#2625]
Hi Guys,

Do you have a way to extends the current RPC methods?, for example I would like to add a harmony RPC method `hmy_getAccountNonce` to return the account nonce.

**Comment by davidmurdoch** [comment]
Not yet. But this is a feature we've been planning on adding.

**Comment by 0xslipk** [comment]
> Not yet. But this is a feature we've been planning on adding.

I would love to do it or help with it. Do you have a branch or spec of what is need it for implement this feature?

**Comment by davidmurdoch** [comment]
Thanks @0xslipk, we don't actually have a spec for this just yet. Would you like me to ping you when we do come up with some concrete steps, hopefully within the next few months?

**Comment by 0xslipk** [comment]
> Thanks @0xslipk, we don't actually have a spec for this just yet. Would you like me to ping you when we do come up with some concrete steps, hopefully within the next few months?

Yeah, I would like that. Let me know when you guys are ready to start and I will help with it.
